packages/container/src/components/SandboxAppHeader.tsx

```diff
@@ -21,10 +21,16 @@ export const SandboxAppHeader: FunctionComponent = () => (
     >
       <ListItemButton
         component="a"
-        href="https://github.com/storyblok/field-plugin/blob/main/README.md"
+        href="https://www.storyblok.com/docs/plugins/field-plugins/introduction"
       >
         Documentation
       </ListItemButton>
+      <ListItemButton
+        component="a"
+        href="https://github.com/storyblok/field-type-examples"
+      >
+        Open-source examples
```

Comment:
```suggestion
        Examples
```

I think "Examples" is enough, because they have to be open source anyway to be shared in public :)